app-admin/kubelet/files/kubelet

```diff
@@ -0,0 +1,31 @@
+#!/bin/bash
+# Wrapper for launching kubelet via rkt-fly stage1
+
+set -e
+
+if [ -z "${KUBELET_VERSION}" ]; then
+    echo "ERROR: must set KUBELET_VERSION"
+    exit 1
+fi
+
+KUBELET_ACI="${KUBELET_ACI:-quay.io/coreos/hyperkube}"
+
+mkdir --parents /etc/kubernetes
+mkdir --parents /var/lib/docker
+mkdir --parents /var/lib/kubelet
+mkdir --parents /run/kubelet
+
+exec /usr/bin/rkt run \
+  --volume etc-kubernetes,kind=host,source=/etc/kubernetes \
+  --volume etc-ssl-certs,kind=host,source=/usr/share/ca-certificates \
+  --volume var-lib-docker,kind=host,source=/var/lib/docker \
+  --volume var-lib-kubelet,kind=host,source=/var/lib/kubelet \
+  --volume run,kind=host,source=/run \
+  --mount volume=etc-kubernetes,target=/etc/kubernetes \
+  --mount volume=etc-ssl-certs,target=/etc/ssl/certs \
+  --mount volume=var-lib-docker,target=/var/lib/docker \
+  --mount volume=var-lib-kubelet,target=/var/lib/kubelet \
+  --mount volume=run,target=/run \
+  $RKT_OPTS \
+  --stage1-image=/usr/share/rkt/stage1-fly.aci \
+  ${KUBELET_ACI}:${KUBELET_VERSION} --exec=/hyperkube -- kubelet "$@"
```

Comment:
Ok, just need to make sure there is a kublet -> hyperkube symlink in the container then.

It is fine if we favor hyperkube for whatever reason, I just don't want what we ship in coreos to be needlessly tied to it when a kublet container is also sufficient. And if the hyperkube container is an exact drop-in replacement for kublet then everything just works no matter what the use selects and no need for extra config goo when simply shipping a symlink means no config goo is required.